build-image/first-boot.bash

```diff
@@ -134,23 +134,17 @@ sleep 10  # Related to: https://github.com/openhab/openhabian/issues/441#issueco
 echo "OK"
 
 echo -n "$(timestamp) [openHABian] Updating repositories and upgrading installed packages... "
-apt-get update &>/dev/null
-/usr/bin/apt-get --yes --fix-broken install &>/dev/null
-apt-get --yes upgrade &>/dev/null
-if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+apt-get --yes --fix-broken install &>/dev/null
+if apt-get --yes upgrade &>/dev/null; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
```

Comment:
This is maybe a question of preference but I find single line `if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi` much easier to read. What do you think?

I would say we can do a file global disable at the top: `# shellcheck disable=SC2181"

Reply:
Hmm. I agree, the old version was more readable. I'm undecided.
It's also true it has a greater risk of running into issues such as if you insert another line before $? is read, effectively making the following line use of your added line's result instead of that of the previous line which is the intention.
And that means to reset gazillion instances which I've already done once.

Reply:
Okey, let's keep it in the "new" pattern. It is actually better programming wise.